### PR TITLE
Cleanup AppVeyor configuration files

### PR DESCRIPTION
**Pull request by escherstair**

### CODE REVIEW

File: tools/appveyor/install.ps1 [hunk 1]
```diff
@@ -3,31 +3,10 @@ $ErrorActionPreference = "Stop"
...
-    #     Remove-Item -Path c:\miktex\texmfs\install\miktex\bin\a5toa4.exe
-    # }
-
-    # Write-Host -ForegroundColor Green "`n### Installing graphviz ###`n"
-    # & cinst --no-progress graphviz
-    # if ($LASTEXITCODE -and $LASTEXITCODE -ne 0) {
-    #     Write-Host -ForegroundColor Red "`n`n*** Installing graphviz failed. Exiting ... ***"
-    #     exit $LASTEXITCODE
-    # }
+    & cinst sphinx --source python
```
Comment (Pro) [inline tools/appveyor/install.ps1]: You probably also need the `sphinx_rtd_theme`

File: tools/appveyor/install.ps1 [hunk 2]
```diff
@@ -3,31 +3,10 @@ $ErrorActionPreference = "Stop"
 try {
     & git submodule --quiet update --init --recursive
 
-    Write-Host -ForegroundColor Green "`n### Installing CMake and python ###`n"
-    & cinst --no-progress cmake python2
-    & C:\Python27\python.exe -m pip install --upgrade pip
-    & C:\Python27\Scripts\pip.exe install six
```
Comment (Pro) [inline tools/appveyor/install.ps1]: Let's see if it runs through, but I guess you need `python-six`

File: tools/appveyor/install.ps1 [hunk 3]
```diff
@@ -3,31 +3,10 @@ $ErrorActionPreference = "Stop"
 try {
     & git submodule --quiet update --init --recursive
 
-    Write-Host -ForegroundColor Green "`n### Installing CMake and python ###`n"
-    & cinst --no-progress cmake python2
```
Comment (Pro) [inline tools/appveyor/install.ps1]: We need both python versions, python2 and python3 since we are testing both. Not sure if they are already preinstalled

Comment (escherstair) [reply tools/appveyor/install.ps1]: Yes, they're both installed

**Comment by escherstair** [comment]
> Also, please squash your commits to clean up the history.

I'm going to do this in the future.
For this PR, could you squash when  you merge?

**Comment by escherstair** [comment]
AppVeyor passed the tests.
Travis doesn't but I think the failure is related to something else.

**Comment by jpfr** [comment]
Squashed and merged.
Thanks for the contribution!

**Pull request merged**